File: fw-portal-web/src/main/resources/static/resources/js/dto/MedidaDto.js
class MedidaDto{

    _tipoMedida;
    _quantidade;

    constructor(tipoMedida,quantidade){
        this._tipoMedida = tipoMedida;
        this._quantidade = quantidade;
    }

    getTipoMedida(){
        return this._tipoMedida;
    }

    getQuantidade(){
        return this._quantidade;
    }
}

File: fw-portal-web/src/main/java/com/faraway/fwportal/dto/MedidaDto.java
package com.faraway.fwportal.dto;

import com.faraway.fwportal.model.domain.Medida;

public class MedidaDto implements Comparable<MedidaDto> {

	private String tipoMedida;

	private Float quantidade;

	public MedidaDto(Medida medida) {
		super();
		this.tipoMedida = medida.getMedida().getNome();
		this.quantidade = medida.getQuantidade();
	}

	public String getTipoMedida() {
		return tipoMedida;
	}

	public Float getQuantidade() {
		return quantidade;
	}

	@Override
	public int compareTo(MedidaDto o) {
		return quantidade.compareTo(o.getQuantidade());
	}

}
